schedule/sle-micro/remote_ssh_controller.yaml

```diff
@@ -14,6 +14,12 @@ conditional_schedule:
         - installation/online_repos
         - installation/installation_mode
         - installation/system_role
+  bls_or_grub:
+    DISTRI:
```

Comment:
What about `microos` with explicit grub2-efi or legacy boot?

Reply:
I don't think its necessary to cover those atm with this specific scenario, but will take into account. Left a note in description of https://progress.opensuse.org/issues/184861

Reply:
> I don't think its necessary to cover those atm with this specific scenario

Currently on x86_64 it uses legacy boot, so it will fail: https://openqa.opensuse.org/tests/5162399

Your VRs are aarch64 only, which are all UEFI

Reply:
Nice! Can we just remove the use of YAML_SCHEDULE there? (I don't like them anyway, they're slowly but surely gaining complexity that's worse than main.pm)